tests/integration/remote/invoke/test_remote_invoke.py

```diff
@@ -0,0 +1,284 @@
+import json
+import uuid
+import base64
+
+from parameterized import parameterized
+
+from tests.integration.remote.invoke.remote_invoke_integ_base import RemoteInvokeIntegBase
+from tests.testing_utils import run_command
+
+from pathlib import Path
+import pytest
+
+
+@pytest.mark.xdist_group(name="sam_remote_invoke_single_lambda_resource")
+class TestSingleResourceInvoke(RemoteInvokeIntegBase):
+    template = Path("template-single-lambda.yaml")
+
+    @classmethod
+    def tearDownClass(cls):
+        # Delete the deployed stack
+        cls.cfn_client.delete_stack(StackName=cls.stack_name)
+
+    @classmethod
+    def setUpClass(cls):
```

Comment:
also for the setup method, should it be moved to the parent class?

Reply:
I inherited this method for each child class to set the `stack-name` based on the class name. This might help debugging failed tests easier.